### ISSUE TITLE
Same thumnail for all videos

### ISSUE THREAD

**Issue by AirplanegoBrr** [ankenyr/jellyfin-youtube-metadata-plugin#117]
All my videos have the same thumbnails.

![image](https://github.com/user-attachments/assets/6b5e595b-1ab1-4f93-8263-8738a977c84f)

I am using TubeSync to download the videos, the files have the following format  `{title}_{source}_({mm}-{dd}-{yyyy})_[{key}].{ext}`

I do have yt-dl on my system.

![image](https://github.com/user-attachments/assets/02b4c7e3-de94-45e4-887f-3b276149c9f3)

#59 says they used JPG and not webp but then never explain how to do so! Thanks @Imroothomelab

#87 also is the same issue. However the fix is "You just don't have youtube-dlp"

That issue mentions trying using the type to Shows and that also didn't work

Here is my TubeSync output

![image](https://github.com/user-attachments/assets/59ded262-2142-438d-b25f-28cd8bdd4ef4)

(This is a SMB share)

I am using Jellyfin 10.10.1

I am going to try and delete all these files and tell TubeSync to download the thumbnail, will update soon.

**Comment by ankenyr** [comment]
Do you have any debug logs I can look at? Do you have debug logging on?
What does the directory structure look like?

**Comment by AirplanegoBrr** [comment]
Do I enable debugging via the `logging.json` jellyfin file?
I see there is an override option. Can I somehow make it log YouTube-metadata only so I don't spam you with unneeded logs?

**Comment by ankenyr** [comment]
Yea I believe that is how you do that. You can also check the official docs
for instructions. On mobile or else I would get a link. There is no way to
filter afaik.

**Comment by ankenyr** [comment]
If you would prefer to send them private my email is in the git logs.

**Comment by AirplanegoBrr** [comment]
Hi! I've emailed u my logs.

**Comment by ankenyr** [comment]
You sure you sent it to me and not someone else who's emails are in the git logs?

**Comment by AirplanegoBrr** [comment]
I've resent the email, the subject is `Github issue #117 logs`

**Comment by AirplanegoBrr** [comment]
Still having this issue. Sent an email on Nov 14
Going to try and downgrade

**Comment by AdamHarrison99** [comment]
@AirplanegoBrr did downgrading fix the issue for you?

I have been having this problem too. It looks like only one thumbnail is being downloaded and used for each Jellyfin "Season". What is even stranger is that older videos I had when the thumbnails were displaying correctly seem to have multiple copies of thumbnails, yet they aren't being used in the web UI. 
![1](https://github.com/user-attachments/assets/67b89f16-7db3-41d2-ae8e-3b95b338a03f)
![1f](https://github.com/user-attachments/assets/244433c6-8850-407b-b30c-1c0573108f60)
![2](https://github.com/user-attachments/assets/b8aac0ec-9830-476f-afec-e74f3be0e2fa)
![2f](https://github.com/user-attachments/assets/0b1a4e6b-4a80-472f-a34d-2ebfb18e6526)

**Comment by AirplanegoBrr** [comment]
> @AirplanegoBrr did downgrading fix the issue for you?
> 
> I have been having this problem too. It looks like only one thumbnail is being downloaded and used for each Jellyfin "Season". What is even stranger is that older videos I had when the thumbnails were displaying correctly seem to have multiple copies of thumbnails, yet they aren't being used in the web UI. 
> ![1](https://github.com/user-attachments/assets/67b89f16-7db3-41d2-ae8e-3b95b338a03f)
> ![1f](https://github.com/user-attachments/assets/244433c6-8850-407b-b30c-1c0573108f60)
> ![2](https://github.com/user-attachments/assets/b8aac0ec-9830-476f-afec-e74f3be0e2fa)
> ![2f](https://github.com/user-attachments/assets/0b1a4e6b-4a80-472f-a34d-2ebfb18e6526)
> I honestly don't know. I downgraded the plugin and got the same results, I removed my library and then created a new library without using this plugin, I use tubesync so it has the thumbnails already.

I created a library with none of my metadata services selected, just nothing data related selected.

I renamed all my thumbnails to have -poster.jpg and boom. They have the correct thumbnails 

Now everything is good and I'm not using this plugin and it looks like my videos are just fine.

I don't even think the plugin is in use as indont have it selected at all.

![Screenshot_20241209-182727_Jellyfin.jpg](https://github.com/user-attachments/assets/2a6b7c27-5e8d-4909-be5e-6d7a395f72a2)

![Screenshot_20241209-182824_Firefox.jpg](https://github.com/user-attachments/assets/b4d48db2-9f43-4e13-84fd-82a28e22fcbd)

Sorry for crappy screenshot

**Comment by tcely** [comment]
I'm hopeful that a release including #120 improves this issue.

**Comment by AKDub** [comment]
So is this fixed in the manual releases? If yes is it OK to install in place of the repository release?

**Comment by tcely** [comment]
Any release that contains `340732397b8d06e622c7a03e0eba68bbb7d4d956` should have improved behavior.

Try it and let us know.

**Comment by AKDub** [comment]
I couldn't get any manually downloaded version to run even after removing the repository release (I admit that is probably due to my lack of ability).

Will just have to wait until there is an actual release or find an alternative if anyone has suggestions.

**Comment by tcely** [comment]
You are welcome to try the files created while I was improving the workflow file.

https://github.com/tcely/jellyfin-youtube-metadata-plugin/releases/tag/1.0.3.10-1-gd3c2d1d-1-g1907b3c-1-g8d6cf2b-1-g2abf072-6-g97a7200-1-g1b3abf8-2-g5a43291-1-gcc5a238-15-gbe38400

**Comment by tcely** [comment]
After #134 was merged, there is now a `.zip` under the release.

MD5: `f23a6d1a76c1c390e4f9f2903fa7460f *youtube-metadata_1.0.3.12.zip`
SHA256: `2a134d8ae953aef9f22108d49b4aa5fca88b7bd8944e1351948852db73219c12 *youtube-metadata_1.0.3.12.zip`

https://github.com/ankenyr/jellyfin-youtube-metadata-plugin/releases/download/1.0.3.10-1-gd3c2d1d-1-g1907b3c-1-g8d6cf2b-1-g2abf072-6-g97a7200-1-g1b3abf8-2-g5a43291-1-gcc5a238-4-g2f97946-3-g0d1ea0b-1-g4d17e66-1-g2ed1d64-1-g531020d-3-g2657c14-1-g5040336/youtube-metadata_1.0.3.12.zip

**Comment by tcely** [comment]
Until the plugin manifest is updated, you will need to install the latest version of the plugin yourself.

https://github.com/ankenyr/jellyfin-plugin-repo/blob/master/manifest.json

It would be great if we could move past `1.0.3.12` soon.

**Comment by tcely** [comment]
Yes. The manifest is pointing to a zip file from 2 years ago, the one I linked to is from 3 weeks ago.
